Formula/go-jsonnet.rb

```diff
@@ -0,0 +1,50 @@
+class GoJsonnet < Formula
+  desc "Go implemention of configuration language for defining JSON data"
+  homepage "https://jsonnet.org/"
+  url "https://github.com/google/go-jsonnet/archive/v0.15.0.tar.gz"
+  sha256 "cddc3462151de4c5c65183b613895575db06297edd12a63e740f09e5e0ed98dd"
+  head "https://github.com/google/go-jsonnet.git"
+
+  depends_on "go" => :build
+
+  conflicts_with "jsonnet", :because => "both install binaries with the same name"
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    bin_path = buildpath/"src/github.com/google/go-jsonnet"
+    contents = Dir["{*,.git,.gitignore}"]
+    bin_path.install contents
+    cd bin_path do
+      system "go", "build", "-o", bin/"jsonnet", "./cmd/jsonnet"
+      # jsonnetfmt will only be added with the next release
+      # system "go", "build", "-o", bin/"jsonnetfmt", "./cmd/jsonnetfmt"
+    end
```

Comment:
maybe this should be sufficient?
```suggestion
      system "go", "build", "-o", bin/"jsonnet", "./cmd/jsonnet"
      # jsonnetfmt will only be added with the next release
      # system "go", "build", "-o", bin/"jsonnetfmt", "./cmd/jsonnetfmt"
```

Reply:
@chenrui333 thank you for your review and feedback, I've just updated PR.

Reply:
Looks good, thanks for the update!!